src/kernel_config_manager.sh

```diff
@@ -131,6 +131,7 @@ function save_config_file()
   if [[ "$ret" == 1 ]]; then
     warning "Warning: $name: there's nothing new in this file"
   elif [[ "$ret" -gt 0 ]]; then
+    git reset --hard # Aborts save operation
```

Comment:
If you are fixing a bug, we need a test that reproduces the issue. However, see my comment on the issue that you created; maybe we can take a different path here.